packages/react-components/react-toast/src/components/ToastContainer/useToastContainerStyles.styles.ts

```diff
@@ -19,6 +20,9 @@ const useStyles = makeStyles({
     pointerEvents: 'all',
     ...shorthands.borderRadius(tokens.borderRadiusMedium),
     '--fui-toast-height': '44px',
+    ...createCustomFocusIndicatorStyle({
+      ...shorthands.outline(tokens.strokeWidthThick, 'solid', tokens.colorStrokeFocus2),
+    }),
```

Comment:
Why not `createFocusOutlineStyle`?

Reply:
because we don't officially support native outline yet

Reply:
Yeah, we disable it by default https://github.com/microsoft/fluentui/blob/90acd97916fbb771d436062cad6a3f85be974e39/packages/react-components/react-tabster/src/focus/createFocusOutlineStyle.ts#L93-L98

Reply:
Ouch, I see. `getFocusOutlineStyles()` does not return styles line `outline*` 😶